app/src/main/java/org/mozilla/fenix/components/toolbar/SwipeRefreshScrollingViewBehavior.kt

```diff
@@ -9,19 +9,21 @@ import android.util.AttributeSet
 import android.view.View
 import androidx.coordinatorlayout.widget.CoordinatorLayout
 import com.google.android.material.appbar.AppBarLayout
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import mozilla.components.concept.engine.EngineView
 import mozilla.components.concept.engine.EngineView.InputResult.INPUT_RESULT_UNHANDLED
 import org.mozilla.fenix.ext.settings
 
 /**
  * ScrollingViewBehavior that will setScrollFlags on BrowserToolbar based on EngineView touch handling
  */
-class SwipeRefreshScrollingViewBehavior(
+class SwipeRefreshScrollingViewBehavior @ExperimentalCoroutinesApi constructor(
     context: Context,
     attrs: AttributeSet?,
     private val engineView: EngineView,
     private val browserToolbarView: BrowserToolbarView
 ) : AppBarLayout.ScrollingViewBehavior(context, attrs) {
+    @ExperimentalCoroutinesApi
```

Comment:
You can now use `@OptIn(ExperimentalCoroutinesApi::class)` and avoid annotating the constructor and anything that creates this class.